src/Symfony/Component/DependencyInjection/Extension/Extension.php

```diff
@@ -82,13 +81,8 @@ public function getConfiguration(array $config, ContainerBuilder $container)
         $namespace = $reflected->getNamespaceName();
 
         $class = $namespace.'\\Configuration';
-        if (class_exists($class)) {
-            $r = new \ReflectionClass($class);
-            $container->addResource(new FileResource($r->getFileName()));
```

Comment:
Understand.. but it's a breakage in behavior.. ie. it doesnt qualify a technical "BC Break".

I'd prefer master+changelog on those, rather then keeping quirky code. Trying to do all.